src/docs/user/references/VariablesReference.adoc

```diff
@@ -51,21 +51,21 @@ The order in this list corresponds to the task life-cycle order.
 | -
 | -
 | <<_control_flow_scripts,flow>> (if)
-| bash, cmd, perl
+| bash, cmd, perl, vbscript
```

Comment:
VBScript (as used above in the doc (in other files)

Reply:
no this is the script engine name, not the wikipedia name